src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.html

```diff
@@ -20,3 +20,25 @@
     {{ role }}{{ !isLast ? ", " : "" }}
   </span>
 </ng-template>
+
+<ng-template #warningTpl
+             let-column="column"
+             let-value="value"
+             let-row="row">
+  <div [ngClass]="getWarningClass(row)"
+       class="border-margin">
+    <div class="warning-content"> {{ value }} </div>
+  </div>
+</ng-template>
+
+<ng-template #durationTpl
+             let-column="column"
+             let-value="value"
+             let-row="row">
+  <i *ngIf="shouldWarn(row)"
+     i18n-title
+     title="User's password is about to expire"
+     class="fa fa-exclamation-triangle"
```

Comment:
You can import icons in user-list.component.ts as `import { Icons } from '~/app/shared/enum/icons.enum';` and use `icons.warning` instead of this. 
See here if you need reference: https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts#L55

Reply:
Thank you for the review. I'll change it.